modules/text2vec-databricks/ent/vectorization_config.go

```diff
@@ -0,0 +1,24 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package ent
+
+type VectorizationConfig struct {
+	Type, Model, ModelVersion, ResourceName string
+	BaseURL                                 string
+	DeploymentID                            string `json:"deploymentId"`
+	ApiVersion                              string
+	IsAzure                                 bool
+	IsThirdPartyProvider                    bool
+	Dimensions                              *int64
```

Comment:
all of those settings and not defined in `class settings` and should be either implemented in `class settings` or removed if not used.

Reply:
removed!